### PR TITLE
Added test cases for GoinGlobal

### PR DESCRIPTION
**Pull request by rjain09**

### CODE REVIEW

**Comment by sak007** [comment]
Sync with main

**Review by sak007 — CHANGES_REQUESTED** [review]
Sync with main

**Comment by wizard-31** [comment]
Sync with main and push the code again

**Review by rjain09 — COMMENTED** [review]
Synced with main

File: test/Scraper/test_oth.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import requests
+
+from importlib.machinery import SourceFileLoader
+goinglobal_scraper = SourceFileLoader('going_global_scraper', 'code/Scraper/going_global_scraper.py').load_module()
+
+def test_get_job_goin_global(mocker):
+    final_result = goinglobal_scraper.get_jobs("Programmer", "", 5, ["Programmer", "Analytic", "Experience"])
+    assert final_result is not None
+
+def test_indeed_scraper_nonworking(mocker):
```
Comment (sak007) [inline test/Scraper/test_oth.py:10]: change function name to goinglobal

**Review by sak007 — APPROVED** [review]

**Review by sak1997 — APPROVED** [review]
Looks fine - good to go

**Pull request merged**